Resources/Prototypes/_Goobstation/GameRules/midround.yml

```diff
@@ -21,7 +21,8 @@
     earliestStart: 30
     weight: 8
     minimumPlayers: 15
-    maxOccurrences: 1
+    maxOccurrences: 3 #Omu change from 1 - better traitors than 5 lings
+    reoccurrenceDelay: 30 #Omu new - able to re-occur every 30 minutes
```

Comment:
This means

Average Player count is 58 so means at max 3 you can roll this 3 times which means at max `TraitorMidround` can select 6 players, due to 2 max
Max Player count means the same due to max occurrences

Omu already suffers an issue with a high amounts of antagonists at max pop resulting in too much chaos which overwhelms our players, this change makes it worse.